errors/handler.go

```diff
@@ -39,7 +44,32 @@ func (handler *KafkaHandler) Handle(instanceID string, err error, data log.Data)
 	}
 
 	data["instance_id"] = instanceID
+	// todo - marshsal error + data to avro encoded message and send to handler.messageProducer.Output()
+
 	log.Error(err, data)
+	eventReport := eventhandler.EventReport{
+		InstanceID: instanceID,
+		EventType:  "error",
+		EventMsg:   err.Error(),
+	}
 
-	// todo - marshsal error + data to avro encoded message and send to handler.messageProducer.Output()
+	producer := eventProducer()
+	avroBytes, err := eventSchema.ReportedEventSchema.Marshal(&eventReport)
+	if err != nil {
+		log.Error(err, nil)
+		return
+	}
+
+	producer.Output() <- avroBytes
+	time.Sleep(time.Duration(5000 * time.Millisecond))
+	producer.Closer() <- true
+
+}
+
+func eventProducer() kafka.Producer {
+	// producerTopic := flag.String("topic", "event-reporter", "producer topic")
+	brokers := []string{"localhost:9092"}
+	producer := kafka.NewProducer(brokers, "event-reporter", int(2000000))
```

Comment:
what is int(2000000)? Should this be a configurable value? It should at least have a meaningful name.